Gems/Atom/RHI/Code/Include/Atom/RHI/DeviceCopyItem.h

```diff
@@ -45,6 +45,9 @@ namespace AZ::RHI
         uint32_t m_sourceOffset = 0;
         uint32_t m_sourceBytesPerRow = 0;
         uint32_t m_sourceBytesPerImage = 0;
+        // The source format is usually same as m_destinationImage's format. When destination image contains more than one aspect,
+        // the format should be compatiable with the aspect of the destination image's subresource
+        Format m_sourceFormat;
```

Comment:
Init to Format::Unknown.

Reply:
Done, also for m_destinatioFormat in the other structs.